clients/venice-admin-tool/src/main/java/com/linkedin/venice/AdminTool.java

```diff
@@ -604,6 +609,28 @@ private static void showSchemas(CommandLine cmd) {
     printObject(valueSchemas);
   }
 
+  private static void executeDataRecovery(CommandLine cmd) {
+    String cluster = getRequiredArgument(cmd, Arg.CLUSTER);
+    String stores = getOptionalArgument(cmd, Arg.STORE);
```

Comment:
There are other parts of this file that seem to get multiple stores in the following manner.

`Set<String> storesToEnforce = Utils.parseCommaSeparatedStringToSet(getRequiredArgument(cmd, Arg.STORE));`

Maybe this would be appropriate here?

Reply:
Thanks Alyssa. I think I can introduce a new `STORES` parameter in admin tool and then reuse the `Utils.parseCommaSeparatedStringToSet` function for parsing it.